### DISCUSSION TITLE
Has Pulsar considered moving from MIT to MPL?

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [pulsar-edit/.github#255] [General]
I don't see why not, they're virtually compatible licenses but MPL is better in nearly all use cases

**Comment by confused-Techie** [comment]
It's not something that's ever been considered as far as I'm aware. Partially it's good to consider that Atom was MIT licensed, which is what Pulsar is a fork of.

Would there be any benefit to relicense the source code of Pulsar to MPL?

**Reply by RustoMCSpit** [reply]
Well MIT and MPL are virtually the same license besides one thing. MPL allows for proprietary usage of the app but the app has to be stored in a separate folder and any changes in that folder must be made public (oversimplification). It's the best of both worlds. You'd get a better understanding if you look it up online for a few minutes

**Reply by Daeraxa** [reply]
They aren't the same though, MPL is copyleft where MIT isn't. MPL may be *weak* copyleft, unlike something like the GPL which makes all code containing GPL code - GPL itself but it still introduces the new requirement to publish the source code.
